app/inline-scripts/instant-globals.js

```diff
@@ -0,0 +1,28 @@
+(function () {
```

Comment:
is this file compiled? or should we only use es5 here?

Reply:
ES5-only